File: src/codes/Users_Manual/geometry_1.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Link;
~using AUTD3Sharp.Utils;
Controller.Builder([
   new AUTD3(new Point3(-AUTD3.DeviceWidth, 0, 0)),
   new AUTD3(Point3.Origin)
])
~;

File: src/codes/Users_Manual/geometry_1.rs
# use autd3::prelude::*;
# 
# #[allow(unused_variables)]
# #[tokio::main]
# async fn main() -> Result<(), Box<dyn std::error::Error>> {
# let _autd = 
Controller::builder([
        AUTD3::new(Point3::new(-AUTD3::DEVICE_WIDTH, 0., 0.)),
        AUTD3::new(Point3::origin())
    ])
#    .open(autd3::link::Nop::builder()).await?;
# Ok(())
# }
